packages/ra-ui-materialui/src/list/DatagridLoading.js

```diff
@@ -106,4 +107,14 @@ const DatagridLoading = ({
     </Table>
 );
 
+DatagridLoading.propTypes = {
+    classes: PropTypes.object,
+    className: PropTypes.string,
+    expand: PropTypes.oneOfType([PropTypes.element, PropTypes.elementType]),
+    hasBulkActions: PropTypes.bool,
+    nbChildren: PropTypes.number,
+    nbFakeLines: PropTypes.number,
+    size: PropTypes.string,
```

Comment:
only 'small' and 'medium' are valid values for this MUI option

Reply:
Done